fluster/main.py

```diff
@@ -47,23 +49,57 @@ class Main:
         self.decoders_dir = DECODERS_DIR
         self.test_suites_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), TEST_SUITES_DIR
-        )
-        # Only use the system directory for test suites if the local directory
-        # doesn't exist and the system directory does exist.
-        if (
-            sys.platform.startswith("linux")
-            and not os.path.exists(self.test_suites_dir)
-            and os.path.exists(TEST_SUITES_DIR_SYS)
-        ):
-            self.test_suites_dir = TEST_SUITES_DIR_SYS
-
+            os.path.dirname(__file__), "..", TEST_SUITES_DIR
+        )
         self.resources_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESOURCES_DIR
+            os.path.dirname(__file__), "..", RESOURCES_DIR
         )
-        self.results_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESULTS_DIR
+        self.results_dir = os.path.join(gettempdir(), RESULTS_DIR)
+
+        is_uninstalled = os.path.exists(
+            os.path.join(os.path.dirname(__file__), "..", ".git")
         )
+        if not is_uninstalled:
+            appname = "fluster"
+            site_data_dir = platformdirs.site_data_dir(appname)
+            user_data_dir = platformdirs.user_data_dir(appname)
+
+            test_suites_dir = os.path.join(site_data_dir, TEST_SUITES_DIR)
+            if os.path.exists(test_suites_dir):
+                self.test_suites_dir = test_suites_dir
+            else:
+                self.test_suites_dir = os.path.join(user_data_dir, TEST_SUITES_DIR)
+
+            resources_dir = os.path.join(site_data_dir, RESOURCES_DIR)
+            if os.path.exists(resources_dir):
+                self.resources_dir = resources_dir
+            else:
+                self.resources_dir = os.path.join(user_data_dir, RESOURCES_DIR)
+
+            print(
+                "NOTE: found a fluster installed version. Using\n"
+                f" * test_suites_dir: {self.test_suites_dir}\n"
+                f" * resources_dir: {self.resources_dir}"
+            )
+
+        else:
+            # Only use the system directory for test suites and resources if the
+            # local directory doesn't exist and the system directory does exist.
+            if (
+                sys.platform.startswith("linux")
+                and not os.path.exists(self.test_suites_dir)
+                and os.path.exists(TEST_SUITES_DIR_SYS)
+            ):
+                line = "=" * 100
+                print(
+                    f"{line}\n"
+                    f'WARNING: Using "{TEST_SUITES_DIR_SYS}" with fluster uninstalled.\n'
+                    "This will be deprecated in future versions.\n"
+                    "Use fluster installed versions instead.\n"
+                    "Check https://github.com/fluendo/fluster/pull/134\n"
+                    f"{line}"
+                )
+                self.test_suites_dir = TEST_SUITES_DIR_SYS
```

Comment:
It seems that none of this is being used, Fluster ends up being initialized with `resources_dir=args.resources` 

https://github.com/fluendo/fluster/blob/13e2d4bb8d2bdcf782cc62abf97ca07afa4a4f63/fluster/main.py#L116-L121

Reply:
It is used as the default value of the argument:

https://github.com/fluendo/fluster/blob/590aad7cb16cd0dcb6cf96b26398f8fd79edc4f2/fluster/main.py#L111-L116